### DISCUSSION TITLE
Adding a toggle in game settings to allow threaded optimization?

### DISCUSSION THREAD

**Discussion by DikanR** [ppy/osu#28427] [Ideas] [answered]
Hi, I have a suggestion.

I have enabled threaded optimization and my FPS increased by few hundreds and latency is reducing from [**0.38ms - 0.60ms**] to [**0.33ms - 0.45ms**] which is a lot for osu! gameplay. But the problem is, threaded optimization setting always reset each restart.

i have read [https://github.com/ppy/osu/discussions/27666](https://github.com/ppy/osu/discussions/27666) and said that you can bypass the issue by enabling compability mode, but it increase my latency to [**0.98ms - 1.10ms**].

i understand that some users is having issues when it enabled so its disabled by default as it mentioned in [https://github.com/ppy/osu/discussions/27666](https://github.com/ppy/osu/discussions/27666).

Can we have it to allowed be enabled without compabilty mode perhaps adding toggle in game to allow threaded optimization? If its not possible please let me know why, thank you.

My osu! version is [20240513.1](https://osu.ppy.sh/home/changelog/stable40/20240513.1)

**Comment by peppy** [comment, marked as answer]
Anything below 1 ms is also not detectable. We set it to off because it reduces *stutters*, which you may actually experience with it on, even if your latency is "better".

**Reply by DikanR** [reply]
Thanks for the reply,

If that's the case, i appreciate the answer. Thank you.
